datahub_sap_hana/ingestion.py

```diff
@@ -221,8 +228,9 @@ def get_column_lineage_view_definitions(
         self, inspector: Inspector
     ) -> Iterable[View]:
         schema: List[str] = inspector.get_schema_names()  # returns a list
-        # TODO, filter schemas
+
         for schema_name in schema:
+
```

Comment:
It isn't clear to me what has changed here, should there be a check against `self.config.schema_pattern`?